Module not found: Error: Can't resolve 'markdown-to-jsx'

Today I started the styleguidist and got an error:

```
ModuleNotFoundError: Module not found: Error: Can't resolve 'markdown-to-jsx' ...
```
This probably happened because of the [markdown-to-jsx package update to 5.3.0](https://github.com/probablyup/markdown-to-jsx/releases)



I was able to fix the problem by installing the package in my project

```
npm i markdown-to-jsx@5.2.0
```

However, this should not be considered as a solution.

------

About my package.json:
```json
"react-styleguidist": "5.1.1",
```


**Does anyone have a similar error?**

--- a/package.json
+++ b/package.json
@@ -58,7 +58,7 @@
     "listify": "^1.0.0",
     "loader-utils": "^1.1.0",
     "lodash": "^4.17.4",
-    "markdown-to-jsx": "^5.2.0",
+    "markdown-to-jsx": "5.2.0",
     "minimist": "^1.2.0",
     "pretty-format": "^19.0.0",
     "prop-types": "^15.5.8",

